Lexicon.md

```diff
@@ -5,10 +5,13 @@
 * __Vulcan Set__: The set of all __Bit Strings__ of infinite length $$ \mathbb{V} := \{ B_0^{\infty}, B_1^{\infty}, \dots, B_{\infty}^{\infty} \} , \ B_i^{\infty} \neq B_j^{\infty} $$.
 * __Information__: Anything that can be encoded into a __Bit String__.
 * __Environment__: __Information__ that is defined relative to a particular __Agent__. It is seperated from an __Agent__ by an __Agent Boundary__, and encompasses all content needed to process any relevant __Actions__ and __Observations__.
+* __World__: Synonymous with __Environment__.
 * __Agent__: A __System__ that interacts with an __Environment__. It can consume zero or more __Observations__, and perform zero or more __Actions__. The decision of which __Actions__ to perform is made based on the __Agent's__ beliefs.
 * __Agent Boundary__: The surface that separates an __Agent__ from its __Environment__.
 * __Observation__: __Information__ that crosses the __Agent Boundary__, entering the __Agent__.
 * __Object__: A set of correlated __Observations__.
 * __System__: An __Object__ composed of correlated member __Objects__.
+* __Model__: A __System__ that represents another __System__. It can be assigned a truth value which expresses the accuracy of the representation. Examples of __Models__ include maps, blueprints, simulations, mathematical models, beliefs and claims.
```

Comment:
Took the liberty of using some phrasing from https://github.com/Vulcan-Academy/Vulcan-Academy.github.io/issues/45#issuecomment-377570599 😃 please lmk if you'd like any of it rephrased.